<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchView</class>
 <widget class="QWidget" name="SearchView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#SearchView{background-color: white;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame { 
background-color: #C3D4E8;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QToolButton" name="toolButtonSearchHeader">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton { 
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #C3D4E8, stop:1 #ECF9FB);

border: none;
}

QToolButton:hover {
background-color: #AFBED0;
}</string>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/resources/system-search2.png</normaloff>:/resources/system-search2.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="labelType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Type your Search here:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit">
        <property name="toolTip">
         <string>the words to search</string>
        </property>
        <property name="whatsThis">
         <string>Here you can enter the words you are searching for.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="toolTip">
         <string>click on this button to run a new search</string>
        </property>
        <property name="whatsThis">
         <string>This button run a new search.</string>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">QFrame#frame_2 { 
background-color: #C3D4E8;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QListWidget" name="friendsList">
        <property name="whatsThis">
         <string>Here you can limit the search to one host.</string>
        </property>
        <item>
         <property name="text">
          <string>Entire network</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Limit Search to:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="toolButtonNetworkHeader">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton { 
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #C3D4E8, stop:1 #ECF9FB);
border: none;
}

QToolButton:hover {
background-color: #AFBED0;
}
</string>
        </property>
        <property name="text">
         <string>Network</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/resources/connections16.png</normaloff>:/resources/connections16.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonSearchHeader</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelType</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonSearchHeader</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonSearchHeader</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonSearchHeader</sender>
   <signal>toggled(bool)</signal>
   <receiver>clearButton</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonNetworkHeader</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonNetworkHeader</sender>
   <signal>toggled(bool)</signal>
   <receiver>friendsList</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
